src/effects/EffectUI.cpp

```diff
@@ -554,12 +554,10 @@ void EffectUIHost::OnApply(wxCommandEvent & evt)
       return;
    }
    
-   // This will take care of calling TransferDataFromWindow() for an effect.
-   if (!mEffectUIHost.GetDefinition().SaveUserPreset(
-      mEffectUIHost.GetCurrentSettingsGroup(), mpAccess->Get()))
-   {
+   if (!TransferDataFromWindow() ||
```

Comment:
If the call to `TransferDataFromWindow()` fails, the call to `SaveUserPreset ` will still be tried? I would not expect that